frontend/public/components/events.jsx

```diff
@@ -64,14 +64,23 @@ const kindFilter = (reference, { involvedObject }) => {
   if (reference === 'all') {
     return true;
   }
-
-  if (isGroupVersionKind(reference)) {
-    const group = apiGroupForReference(reference);
-    const kind = kindForReference(reference);
-    const { group: eventGroup } = groupVersionFor(involvedObject.apiVersion);
-    return involvedObject.kind === kind && eventGroup === group;
-  }
-  return involvedObject.kind === reference;
+  let filterKind = false;
```

Comment:
I would change the impl here from `let filterKind =false` to a `const` and use `_.filter` or a better util than `_.map`.

Reply:
I'm not super worried about the form tbh, this is reasonable as a bug fix.  Perhaps we can throw a unit test up for it quick?